.github/workflows/contract-integration.yaml

```diff
@@ -20,16 +20,42 @@ jobs:
           python-version: 3
       - run: |
           pip install -e .
-          cd provider/*project
+          cd lookoutvision/*project
           curl -s -L https://github.com/aws/aws-sam-cli/releases/latest/download/aws-sam-cli-linux-x86_64.zip -o sam.zip
           unzip -q sam.zip -d sam-install
           sudo ./sam-install/install
           sam local start-lambda &
-          pip install --quiet cloudformation-cli-java-plugin
+          pip install --quiet cloudformation-cli-java-plugin cloudformation-cli-python-plugin
           mvn package --no-transfer-progress -Dmaven.test.skip=true
           cfn test --enforce-timeout 240
           ! grep -q -E -i '(accessKeyId|secretAccessKey|sessionToken)' rpdk.log*
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           AWS_REGION: us-east-1
+  contract-integration-python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: aws-cloudformation/aws-cloudformation-resource-providers-frauddetector
+          path: frauddetector
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3
+      - run: |
+          pip install -e .
+          cd frauddetector/*outcome
+          curl -s -L https://github.com/aws/aws-sam-cli/releases/latest/download/aws-sam-cli-linux-x86_64.zip -o sam.zip
+          unzip -q sam.zip -d sam-install
+          sudo ./sam-install/install
+          sam local start-lambda &
+          pip install --quiet cloudformation-cli-java-plugin cloudformation-cli-python-plugin
```

Comment:
I see this uses `actions/setup-python@v2`, and not `actions/setup-java@v1`. So, does this need to `pip install ... cloudformation-cli-java-plugin` at all?